### PR TITLE
Add filters for Audit, Scan & Docker Scan

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/jfrog-cli#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----

### CODE REVIEW

File: utils/cliutils/commandsflags.go [hunk 1]
```diff
@@ -1283,6 +1284,14 @@ var flagsMap = map[string]cli.Flag{
 		Name:  RequirementsFile,
 		Usage: "[Optional] [Pip] Defines pip requirements file name. For example: 'requirements.txt' ` `",
 	},
+	WithFixVersion: cli.BoolFlag{
+		Name:  WithFixVersion,
+		Usage: "[Optional] Set to true if you wish to display the audit results with fix version only ` `",
```
Comment (eyalbe4) [inline utils/cliutils/commandsflags.go:1289]: ...display the audit results with fix version only...
-->
...display issues which have a fix version only

File: utils/cliutils/commandsflags.go [hunk 2]
```diff
@@ -1283,6 +1284,14 @@ var flagsMap = map[string]cli.Flag{
...
+	},
+	MinSeverity: cli.StringFlag{
+		Name:  MinSeverity,
+		Usage: "[Optional] Set the minimum severity to display results and filter the rest ` `",
```
Comment (eyalbe4) [inline utils/cliutils/commandsflags.go:1293]: Set the minimum severity to display results and filter the rest
-->
Set the minimum severity of issues to display. Set one following values are accepted: Low, Medium, High or Critical

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
* Let's change the flags names as we discussed.
* Let's update the documentation.

File: utils/cliutils/commandsflags.go [hunk 3]
```diff
@@ -1833,10 +1842,10 @@ var commandFlags = map[string][]string{
...
 		xrUrl, user, password, accessToken, serverId, specFlag, threads, scanRecursive, scanRegexp, scanAnt,
-		project, watches, repoPath, licenses, xrOutput, fail, ExtendedTable, BypassArchiveLimits,
+		project, watches, repoPath, licenses, xrOutput, fail, ExtendedTable, BypassArchiveLimits, MinSeverity, FixableOnly,
 	},
 	DockerScan: {
-		serverId, project, watches, repoPath, licenses, xrOutput, fail, ExtendedTable, BypassArchiveLimits,
+		serverId, project, watches, repoPath, licenses, xrOutput, fail, ExtendedTable, BypassArchiveLimits, MinSeverity, FixableOnly,
 	},
 	BuildScan: {
 		xrUrl, user, password, accessToken, serverId, project, vuln, xrOutput, fail, ExtendedTable, rescan,
```
Comment (sverdlov93) [inline utils/cliutils/commandsflags.go:1851]: build scan?

File: scan/cli.go [hunk 4]
```diff
@@ -350,7 +354,9 @@ func DockerScan(c *cli.Context, image string) error {
 		SetIncludeLicenses(c.Bool("licenses")).
 		SetFail(c.BoolT("fail")).
 		SetPrintExtendedTable(c.Bool(cliutils.ExtendedTable)).
-		SetBypassArchiveLimits(c.Bool(cliutils.BypassArchiveLimits))
+		SetBypassArchiveLimits(c.Bool(cliutils.BypassArchiveLimits)).
+		SetFixableOnly(c.Bool(cliutils.FixableOnly)).
+		SetMinSeverityFilter(c.String(cliutils.MinSeverity))
```
Comment (sverdlov93) [inline scan/cli.go:359]: It's better to validate the MinSeverityFilter like we do in format:
format, err := commandsutils.GetXrayOutputFormat(c.String("format"))

File: utils/cliutils/commandsflags.go (hunk 3 above)
Comment (omerzi) [reply utils/cliutils/commandsflags.go:1851]: done

File: scan/cli.go (hunk 4 above)
Comment (omerzi) [reply scan/cli.go]: done

**Pull request merged**